test_util/test_upgrade_vpc.py

```diff
@@ -235,6 +247,13 @@ def setup_cluster_workload(self, dcos_api: DcosApiSession, healthcheck_app: dict
             for app in test_apps:
                 assert app['instances'] == len(self.tasks_start[app['id']])
 
+            # Save the master's state of the task to compare with
+            # the master's view after the upgrade.
+            # See this issue for why we check for a difference:
+            # https://issues.apache.org/jira/browse/MESOS-1718
+            self.task_state_start = self.get_master_task_state(dcos_api, self.tasks_start[0])
+            log.debug('Test app state at start:\n' + pprint.pformat(self.task_state_start))
```

Comment:
My preference would be to not blindly log some data without any context.

Alternate recommendations (in my prefered order):
1. Remove this and include the state comparison in the assertion, so that when the test fails it tells you exactly why it failed (rather than having to possibly scroll back hundreds of log lines)
2. Move this log statement into the test method since that will at least keep it inside the context of the test that cares about it